imjoy/imjoyPluginEngine.py

```diff
@@ -1654,8 +1657,9 @@ def launch_plugin(
             "Failed to setup plugin virual environment or its requirements: " + str(e),
             type="error",
         )
+        error = e
```

Comment:
It looks weird that we don't take further action here, upon error. That's why I added this error variable, which we check below and abort if it's truthy. Not sure if that's the correct way of doing things.

Reply:
I think you can do `abort.set()` instead of introducing `error`.

Reply:
Ok.